src/js/view/controls/components/custom-button.js

```diff
@@ -15,7 +15,9 @@ export default class CustomButton {
         }
 
         let iconElement;
-        if (img && img.substring(0, 4) === '<svg') {
+        if (img instanceof SVGSVGElement) {
```

Comment:
Is `SVGSVGElement` a thing? or should it be `SVGElement`?

Reply:
It is thing which also returns true, but it should be SVGElement